lib/tessel/provision.js

```diff
@@ -256,7 +287,7 @@ function AlreadyAuthenticatedError() {
 util.inherits(AlreadyAuthenticatedError, Error);
 
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
-
+module.exports.TESSEL_AUTH_KEY = filepath;
```

Comment:
@rwaldron the reason we have the default filepath exported is so that it can be used as the default key on line 42 of bin/tessel-2.js (see above). I agree that it's kind of hacky though. Do you have another suggestion?